lib/cartopy/tests/mpl/test_ticker.py

```diff
@@ -84,7 +89,7 @@ def test_LatitudeFormatter_degree_symbol():
 
 
 def test_LatitudeFormatter_number_format():
-    formatter = LatitudeFormatter(number_format='.2f')
+    formatter = LatitudeFormatter(number_format='.2f', decimal=True)
```

Comment:
This and similar tests may have to be reverted depending on action taken re: breaking changes to Longitude and Latitude Formatters.